flowzone.yml

```diff
@@ -668,6 +670,12 @@ on:
       CF_API_TOKEN:
         description: "Cloudflare API token with limited access for Pages projects"
         required: false
+      PYPI_TOKEN:
+        description: "Token to access pypi.org Python packages."
+        required: false
+      PYPI_TEST_TOKEN:
+        description: "TODO: do we need both PyPi tokens?"
+        required: false
```

Comment:
Did you add these or did the linter?

Probably best to add the lint action in one commit, and perform the lint in another commit for visibility.

Reply:
Yeah, that was added by me, because the linter picked up the missing secrets. Perhaps you know if we need both?